### ISSUE TITLE
Clarify Crypto Listing Account Field

### ISSUE THREAD

**Issue by jjeffryes** [OpenBazaar/openbazaar-desktop#1414]
On cryptocurrency listings, not all coins have addresses. Some use account names instead, such as Steem, and Bitshares. See https://blocktrades.us/ for an example.

We should make it more clear that the address field on a crypto purchase is for whatever identifier is needed for that coin.

![image](https://user-images.githubusercontent.com/1584275/41266403-bbd3ed8e-6dc4-11e8-9e55-9a249875327c.png)

**Comment by morebrownies** [comment]
Good point. There may be some variance in what each coin requires to send/receive funds. Buying Steem for example may require the buyer to send their `username` on Steem. We can patch that up specifically for Steem by changing the copy of label input placeholder, but I'm not sure how friendly that will play with our system.

`Steem Account Name` label would be an appropriate change in this scenario, with input placeholder text "Paste your Steem account name"

![screenshot 2018-06-11 20 13 29](https://user-images.githubusercontent.com/13066886/41268287-3512c7b2-6db4-11e8-9941-5ab322beb967.png)

**Comment by morebrownies** [comment]
> hmm... I'm really not sure how scalable it is to have a specific label per currency given that we're planning on have 1500+ of them.

To be clear, I'm not suggesting a unique label/placeholder for every coin. There are one-off scenarios that are absolutely unique, but it doesn't exist across all 1500+ coins.

We could try going with a more generic approach, such as: 'Your [coin] address (or username)` but that may still lead to user error. I would be ok with going the simpler route first, and if it's still leading to a lot of confusion, we may have to invest more effort into it.

> I don't want too seem harsh, but there also has to be some responsibility on the user to understand the nuances of what they are buying.

Sure. We haven't heard of this being a widespread issue yet, and the Vendor could reject orders with in-proper information, but we should definitely keep an eye on this.

**Comment by jjeffryes** [comment]
I think a more generic label with helper text would work. 

Maybe "Transfer [coin] To:" as the label, and "The address or other identifier the seller will user to transfer [coin] to you."

**Comment by christroutner** [comment]
Perhaps a 'Are you sure you got the address right' modal is appropriate? Something with a disclaimer saying that if they put in the address incorrectly, their funds could be lost.

**Comment by morebrownies** [comment]
> Maybe "Transfer [coin] To:" as the label, and "The address or other identifier the seller will user to transfer [coin] to you."

I would be fine with this approach @jjeffryes. Starting simple is better here. We can revisit if needed.

**Comment by morebrownies** [comment]
> I'm thinking of an additional modal that pops up when the user clicks the Pay button. Just an idea. Maybe not a good one.

@christroutner let's see if the simple copy change makes a difference in the short term. If not effective, we may have to be more in the user's face and do things like modals and what not. We'll see.

**Comment by rmisio** [comment]
@jjeffryes @morebrownies Would `Receive [coin] At:` be better than `Transfer [coin] To:` since this is in the purchase screen and is from the perspective of the buyer?

**Comment by jjeffryes** [comment]
That makes sense to me.
